documentation/asciidoc/computers/camera/rpicam_options_common.adoc

```diff
@@ -1,29 +1,33 @@
-=== Common command line options
+=== Common options
 
 The following options apply across all the `rpicam-apps` with similar or identical semantics, unless otherwise noted.
 
-----
-	--help,		-h		Print help information for the application
-----
+To pass one of the following options to an application, prefix the option name with `--`. If the option requires a value, pass the value immediately after the option name, separated by a single space. If the value contains a space, surround the value in quotes.
 
-The `--help` option causes every application to print its full set of command line options with a brief synopsis of each, and then quit.
+Some options have shorthand aliases, for example `-h` instead of `--help`. Use these shorthand aliases instead of the full option name to save space and time at the expense of readability.
 
-----
-	--version			Print out a software version number
-----
+==== `help`
+
+Alias: `-h`
+
+Prints the full set of options, along with a brief synopsis of each option. Does not accept a value.
+
+==== `version`
+
+Prints out version strings for `libcamera` and `rpicam-apps`. Does not accept a value.
 
-All `rpicam-apps` will, when they see the `--version` option, print out a version string both for `libcamera` and `rpicam-apps` and then quit, for example:
+For example:
```

Comment:
Perhaps "Example output:" here would be more explanatory than "For example:" ? :man_shrugging: